index.ios.js

```diff
@@ -29,27 +18,18 @@ const store = createStore(
   applyMiddleware(thunk, promise, logger)
 );
 
-class findAR extends Component {
-
-  constructor() {
-    super();
-  }
+const scenes = routerActions.create(
+  <Scene key="root" hideNavBar>
+    <Scene key="ar" component={ARView} />
+    <Scene initial key="map" component={MapView} />
+    <Scene key="list" component={PinListView} />
```

Comment:
we also need to add scenes for the other views:
settings page (todo) scenes like these should be routed when you press on the side menu!
log in page(in the process), this should routed first before user sees any other routes.